projects/gloo/pkg/api/external/envoy/extensions/transformation_ee/transformation.pb.go

```diff
@@ -367,13 +367,19 @@ type Action struct {
 	// Used mostly to help ID specific actions in logs.
 	// If left null will default to unknown
 	Name string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
+	// Deprecated in favor of DlpMatcher
```

Comment:
```suggestion
	// Deprecated in favor of DlpMatcher.
```
since the comments are all put together on one line when rendered